src/Middleware/RequestBodyBufferMiddleware.php

```diff
@@ -29,27 +30,23 @@ public function __construct($sizeLimit = null)
 
     public function __invoke(ServerRequestInterface $request, $stack)
     {
-        $size = $request->getBody()->getSize();
-
-        if ($size === null) {
-            return new Response(411, array('Content-Type' => 'text/plain'), 'No Content-Length header given');
-        }
-
-        if ($size > $this->sizeLimit) {
-            return new Response(413, array('Content-Type' => 'text/plain'), 'Request body exceeds allowed limit');
-        }
```

Comment:
We should probably keep this check: it makes sense to fail early in case the client is trying to upload a gigabyte file over a slow uplink :-)

Reply:
Fixed. Please forgive me for force-pushing. I didn't realize you were already reviewing.

Reply:
No worries, happy to see you're faster with coding than I am with reviewing :shipit: